database/migrations/2018_06_04_125400_create_process_files_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateProcessFilesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('PROCESS_FILES', function(Blueprint $table)
```

Comment:
Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.